.drone.yml

```diff
@@ -0,0 +1,85 @@
+workspace:
+  base: /drone
+  path: src/github.com/owncloud/client
+
+branches:
+  - master
+  - 2.4
+  - 2.3
```

Comment:
I don't know what this "branches" entry is for exactly, but we don't need 2.3 as this branch is dead.
(But there will be a 2.5 soon)

Reply:
It's similar to the `branches` attribute in Travis, it limits builds to these target branches, so it matches pushs to these branches as well as pull requests targetting these branches.